### DISCUSSION TITLE
small help

### DISCUSSION THREAD

**Discussion by Paceski** [games647/FastLogin#890] [General]
![image](https://user-images.githubusercontent.com/82279522/184627503-1682572b-f5c8-4aa6-a3cd-d596cdc6cf55.png)

Hey, I'm wondering if this message can be disabled for premium logins and only be shown for cracked people?

**Comment by Paceski** [comment]
I am using https://dev.bukkit.org/projects/loginsecurity

**Comment by ApliNi** [comment]
Maybe you can find a plugin with an anvil GUI for working with login plugins and delete those messages. 
https://www.spigotmc.org/resources/anvillogin.31861/

**Comment by TuxCoding** [comment]
This all depends on your auth plugin. Auth plugins needs an event before printing this message, so we could prevent it. Otherwise we could install a message filter, but it's difficult, could cause compatibility issues and requires a lot of effort.

**Reply by ApliNi** [reply]
I don't have this problem when I use AuthMe, AuthMe can delay the display of the login message, this time is enough for FastLogin to log in genuine users. )
